deploy/dev/clickhouse-operator-install-yaml-template-04-section-deployment-with-configmap.yaml

```diff
@@ -46,7 +46,7 @@ spec:
       containers:
         - name: clickhouse-operator
           image: ${OPERATOR_IMAGE}
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
That's not correct. We need an Always policy since the same tag is ofter re-deployed during development

Reply:
I'll let you and @vzakaznikov have a fight here :)

Reply:
Isn't redeploying the same tag again and again a bad practice?
E.g. when I refer to 21.8, I want the exact same 21.8 that I used to have yesterday. If I want a newer one - I will refer to 21.8.1.3

Reply:
Development version -- yes. E.g. 0.16.0 tag is re-deployed after every commit automatically, until we release it.

Reply:
Well, I can add something like pre-downloading image before launch in docker and propagate it all the way down to minikube. I'll do it tomorrow